apps/extension/src/wallet-services.ts

```diff
@@ -29,7 +57,13 @@ export const startWalletServices = () =>
           case false:
             return false;
           case ServicesMessage.ClearCache:
-            void services.clearCache().finally(() => respond());
+            void (async () => {
+              blockProcessor.stop('clearCache');
+              await Promise.allSettled([localExtStorage.remove('params'), indexedDb.clear()]);
```

Comment:
If this throws an error, it seems to be swallowed right? Is there a way to respond with an error?

Reply:
an error here will throw to console, which is about all we can do anyway, as the calling location isn't set up to handle an error response. the extension will restart regardless. an error happening in this case is quite catastrophic and there is no clear way to handle it aside from reinstalling, so i think this is fine.